apis/project.cattle.io/v3/app_types.go

```diff
@@ -58,7 +58,7 @@ type AppStatus struct {
 	Notes                string            `json:"notes,omitempty"`
 	Conditions           []AppCondition    `json:"conditions,omitempty"`
 	LastAppliedTemplates string            `json:"lastAppliedTemplate,omitempty"`
-	HelmVersion          string            `json:"helmVersion,omitempty" norman:"nocreate,noupdate"`
```

Comment:
Same thing?

Reply:
same thing, just wanted it to be in the same order as the other ones I added 🤷‍♂